deployment/arm/cosmosdb.json

```diff
@@ -0,0 +1,85 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
Can the database / collections also be deploy as part of the template? or will we rely on the Entity framework to handle this for us?

Reply:
I have designed the backend api app to create the database and collection if it doesn`t exists.